log4j-1.2-api/src/main/java/org/apache/log4j/builders/layout/TTCCLayoutBuilder.java

```diff
@@ -91,10 +91,10 @@ public Layout parseLayout(Element layoutElement, XmlConfiguration config) {
 
     @Override
     public Layout parseLayout(PropertiesConfiguration config) {
-        boolean threadPrinting = getBooleanProperty(THREAD_PRINTING_PARAM);
-        boolean categoryPrefixing = getBooleanProperty(CATEGORY_PREFIXING_PARAM);
-        boolean contextPrinting = getBooleanProperty(CONTEXT_PRINTING_PARAM);
-        String dateFormat = getProperty(DATE_FORMAT_PARAM);
+        boolean threadPrinting = getBooleanProperty(THREAD_PRINTING_PARAM, true);
```

Comment:
I can see this change matches what the defaults are indeed in Log4j 1.2 (true values) but we need a test that shows that this change works.
TY!

Reply:
I added a test, which was failing before.